prvocisla.cpp

```diff
@@ -49,20 +52,26 @@ int * sieve(int n){
 
 void printPrime(int* L, int n) {
     int j = 0;
+
     while (L[j] <= n){
         cout << L[j] << " ";
         j++;
     }
+
     cout << endl;
 }
 
 int main()
 {
     int n = 200;
+
     int * L;
+
     L = sieve(n);
```

Comment:
Problem s navracenim poctu prvku v poli (pokud nechces to zakoncovaci cislo) se da resit napr takto:
```suggestion
    int numPrime = 0; // vzdy je dobre mit s deklaraci i definici = zapsani hodnoty
    L = sieve(n, &numPrime);
```